kokoro/linux/dockerfile/test/php_32bit/Dockerfile

```diff
@@ -0,0 +1,224 @@
+FROM 32bit/debian:jessie
```

Comment:
is it better to have all PHP versions setup in one file? or would it be preferable split them up into separate Dockerfiles?

Reply:
As discussed on meeting, for php, we can share docker image as much as possible.
However, it's not possible to share docker image between 64 and 32 bit tests.